private/pkg/spdx/spdx.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spdx
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"sort"
+	"strings"
+
+	"go.uber.org/multierr"
+)
+
+const (
+	licenseListURL = "https://raw.githubusercontent.com/spdx/license-list-data/v3.12/json/licenses.json"
```

Comment:
If you update to the latest file: https://raw.githubusercontent.com/spdx/license-list-data/v3.18/json/licenses.json

The full URL is now in the `reference` field.